src/components/welcomeButton/styles.js

```diff
@@ -4,7 +4,9 @@ import { colors, fonts } from '../../theme';
 export const styles = StyleSheet.create({
   buttonView: {
     flex: 1,
-    justifyContent: 'center',
+  },
+  touchable: {
+    width: '100%',
```

Comment:
This can may enable clicking outside the button, on the left/right. We should check and if that's the case we should make the view full width, delete unnecessary internal view and add all the style of that view into the touchable itself.